tests/conftest.py

```diff
@@ -84,3 +114,15 @@ def user_books(django_db_setup, django_db_blocker, books, user):
             for book in books
         ]
         return UserBook.objects.bulk_create(user_books)
+
+
+@pytest.fixture(scope="module")
+def user_fav_books(django_db_setup, django_db_blocker, two_books, user):
+    with django_db_blocker.unblock():
+        statuses = cycle('r c'.split())
```

Comment:
Was going to say just on completed but double-checking, you can favorite books in any tab, so this is a good test

Reply:
cool